tcms/testcases/tests/test_views.py

```diff
@@ -64,6 +64,33 @@ def test_user_in_default_group_sees_comments(self):
                 html=False
             )
 
+    def test_user_sees_bugs(self):
+        initiate_user_with_default_setups(self.tester)
+
+        url = reverse('caserun-detail-pane', args=[self.case_run_1.case_id])
+        response = self.client.get(
+            url,
+            {
+                'case_run_id': self.case_run_1.pk,
+                'case_text_version': self.case_run_1.case.latest_text_version()
+            }
+        )
+
+        self.assertEqual(HTTPStatus.OK, response.status_code)
+
+        self.case_run_1.add_bug(1, 1)
+        self.case_run_1.add_bug(2, 2)
```

Comment:
Move this before `self.client.get()` - you want bugs to be added to the TCR before you fetch the HTML details.

Also use `BugFactory` to create the bug objects instead of hard-coding the IDs.